src/map/npc.c

```diff
@@ -4813,8 +4860,10 @@ int npc_reload(void) {
 	for (bl = mapit->first(iter); mapit->exists(iter); bl = mapit->next(iter)) {
 		switch(bl->type) {
 			case BL_NPC:
-				if( bl->id != npc->fake_nd->bl.id )// don't remove fake_nd
+				if (bl->id != npc->fake_nd->bl.id ) {// don't remove fake_nd
 					npc->unload(BL_UCAST(BL_NPC, bl), false);
+					status->change_clear(bl, 1);
```

Comment:
here again look like you introduced npc statuses. This is generic thing and not related to clans.
This mean better move generic one code about npc statuses into separate commit